packages/openapi-code-generator/src/test/unit-test-inputs-3.0.3.yaml

```diff
@@ -168,9 +168,6 @@ components:
 
     AdditionalPropertiesSchema:
       type: object
-      properties:
```

Comment:
This actually created a typescript error, as the type of the index property must include all named property types.

(discovered after adding typescript compilation checks to the test suite)

![image](https://github.com/user-attachments/assets/f2a3727b-9276-486e-82b4-3727ca8a29a5)